services/QuillLMS/app/workers/admin_diagnostic_reports/diagnostic_overview_worker.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module AdminDiagnosticReports
```

Comment:
This is basically a copy of `Snapshots::CacheSnapshotTopXWorker` with tweaks to `PUSHER_EVENT` and `QUERIES`.  I mentioned a planned refactor to unify Controller logic in a parent, and I intend to do the same thing with these workers in that refactor.